CHANGES.rst

```diff
@@ -37,7 +37,8 @@ There's a frood who really knows where his towel is.
 - Fix various exceptions raised when content has Archetypes based lead images.
   [hvelarde]
 
-
+- Fix CSS error in logo position.
```

Comment:
almost :-) note that changelog entries should be inserted on top of the others; always leave a blank line between changelog entries.

please read the [Plone styleguides](https://docs.plone.org/develop/styleguide/index.html) to understand a little bit more how we work.

next time create a branch on this repo so I can modify the PR in case needed.